plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -690,6 +705,32 @@ public void testBigQueryExternalTable()
         }
     }
 
+    @Test
+    public void testBigLakeExternalTable()
+    {
+        String biglakeExternalTable = "test_biglake_external" + randomNameSuffix();
+        try {
+            onBigQuery("CREATE EXTERNAL TABLE test." + biglakeExternalTable +
+                    " WITH CONNECTION `" + bigQueryConnectionId + "`" +
+                    " OPTIONS (format = 'CSV', uris = ['gs://" + gcpStorageBucket + "/tpch/tiny/region.csv'])");
+            assertQuery("SELECT table_type FROM information_schema.tables WHERE table_schema = 'test' AND table_name = '" +
+                    biglakeExternalTable + "'", "VALUES 'BASE TABLE'");
```

Comment:
We could use `assertThat(getQueryRunner().tableExists(getSession(), ...))` instead. 
Same for L723-724.

Reply:
Done. I had to set the schema to `TEST_SCHEMA` in BigQueryRunner for this to work. Hope that's OK.